objects/src/transaction/tx_witness.rs

```diff
@@ -72,7 +73,12 @@ impl TransactionWitness {
 
     /// Returns a reference to the transaction script.
     pub fn tx_script(&self) -> Option<&TransactionScript> {
-        self.tx_script.as_ref()
+        self.tx_args.tx_script()
+    }
+
+    /// Returns a reference to the transaction script.
+    pub fn tx_args(&self) -> Option<&BTreeMap<NoteId, Word>> {
+        self.tx_args.note_args()
     }
```

Comment:
The name and comment `tx_args()` don't match the return type - but also, should we just have a single accessor for `TransactionArgs` instead of these two accessors? i.e., something like:
```Rust
pub fn tx_args(&self) -> &TransactionArgs
```

Reply:
Indeed that slipped. Let me change